### ISSUE TITLE
Global variable `event: Event` in lib.es6.d.ts

### ISSUE THREAD

**Issue by sod** [Microsoft/TypeScript#11583]
**TypeScript Version:**  2.0.3

**Code**

tsconfig.json includes lib `es6`:

``` json
{
    "compilerOptions": {
        "lib": [
            "es6",
            "dom"
        ]
    },
}
```

``` ts
function eventHandler() {
    // typescript doesn't complain about missing event variable, because event is declared as global in lib.es6.d.ts
    console.log(event.target);
}
```

**Expected behavior:**

Don't expect event to exist on the global object. Couldn't reproduce it in Safari, Firefox or Chrome. It's always `console.log(window.event); // undefined`

**Actual behavior:** 

`event: Event` is declared as global variable in lib.es6.d.ts
https://github.com/Microsoft/TypeScript/blob/2e7c83abc833534181d97c534bfac865cd93b1f0/lib/lib.es6.d.ts#L18680
https://github.com/Microsoft/TypeScript/blob/2e7c83abc833534181d97c534bfac865cd93b1f0/lib/lib.es6.d.ts#L20129

**Comment by kitsonk** [comment]
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/event) it is non-standard IE/Edge property.  The typings for the `lib` are derived from the Edge IDL, which is likely that caused it to be there.

But it does seem to be present in other browsers (I checked Chrome)

```
'event' in window; // returns `true`
'foo' in window; // returns `false`
```

I guess it should be properly typed as `Event | undefined`.

**Comment by sod** [comment]
Indeed. You are right. The property exists on window. But it is `undefined`, even in Edge 13 & IE 11.

![image](https://cloud.githubusercontent.com/assets/905328/19345741/26e652f0-9140-11e6-8d10-a1e4862fdc68.png)

So maybe `declare var event: undefined`, without the `Event`.

**Comment by kitsonk** [comment]
Yes, sorry, I meant to say that.

I suspect that IE/Edge continues to support it, though it isn't part of standards, and other browsers define it, but keep its value always `undefined` to avoid breaking lots of *bad* code, that running in strict mode, might cause an exception.

Anyways, I still suspect it is better to have it as `Event | undefined`.

**Comment by mhegazy** [comment]
PRs welcomed. You can find more information about contributing lib.d.ts fixes at https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes.

**Comment by jarrydfillmore** [comment]
Awesome!

**Comment by alisabzevari** [comment]
I will do it :smile:
